bulbs/content/models.py

```diff
@@ -616,7 +616,7 @@ def delete_from_instant_article_api(sender, instance=None, **kwargs):
                     fb_token_path))
             return
 
-        fb_access_token = vault.read(fb_token_path)['authtoken']
+        fb_access_token = vault.read(fb_token_path).get('authtoken', None)
```

Comment:
If you do this, then you need to check for `None` and raise. 
Also you don't need `, None`, this is the default and makes for cleaner code
```
     fb_access_token = vault.read(fb_token_path).get('authtoken')
```